backend/api/metrics_views.py

```
@@ -297,6 +301,123 @@ def get(self, request):
         return Response({'data': data})
 
 
+class CommunityContributionMetricsView(APIView):
+    """
+    Lean metrics-page view for community contribution rows and totals.
+
+    The generic /contributions/ endpoint serializes nested user/type/evidence
+    payloads for explorer pages. Metrics only needs the table fields below plus
+    aggregate totals, so this view keeps the query to scalar values.
+    """
+    permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)